### PR TITLE
Add missing column "fk_hearing_defendant_id" on hearing_outcome_aud

### PR DESCRIPTION
**Pull request by johnyfarrar**

### CODE REVIEW

File: src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE if exists hearing_outcome_aud ADD COLUMN fk_hearing_defendant_id integer ;
+
+END;
```
Comment (AndySweilam) [inline src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql:5]: ALTER TABLE hearing_outcome_aud
    ADD COLUMN IF NOT EXISTS id;
    
    ALTER TABLE hearing_outcome_aud 
    ADD CONSTRAINT fk_hearing_defendant_id FOREIGN KEY (id) REFERENCES {something?};

Comment (AndySweilam) [reply src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql:5]: If that column already exists ID, then ignore the first statement

Comment (AndySweilam) [reply src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql:5]: This is assuming the column name is 'id'

File: src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE if exists hearing_outcome_aud ADD COLUMN fk_hearing_defendant_id integer;
```
Comment (Obsiye) [inline src/main/resources/db/migration/courtcase/V2176__add_audit_column_for_hearing_outcome_fk_hearing_defendant_id.sql:3]: Hi, for my own curiosity, what is `if exists` checking ? is it checking that the table `hearing_outcome_aud` exists or the column `fk_hearing_defendant_id`?

If it's checking the `fk_hearing_defendant_id` then maybe something like

`ALTER TABLE hearing_outcome_aud ADD COLUMN IF NOT EXISTS fk_hearing_defendant_id integer;`

**Review by Obsiye — DISMISSED** [review]

**Review by Obsiye — APPROVED** [review]

**Pull request merged**